packages/utils/test/format.test.js

```diff
@@ -57,3 +57,7 @@ test('parseNearAmount fails when parsing values with ≥25 decimal places', () =
         'Cannot parse \'0.0000080990999998370878871\' as NEAR amount'
     );
 });
+
+test('NEAR_NOMINATION value', () => {
+    expect(NEAR_NOMINATION).toEqual(1000000000000000000000000n)
```

Comment:
I see here we added the test for `NEAR_NOMINATION`, to make sure that the precision is right.

What puzzles me is that, just above this, there is a test checking that `formatNearAmount` is compatible with `parseNearAmount`. How is that it didn't fail when the bug of `BigInt(10**24)` was introduced?

They all need to be reviewed and, if necessary, more tests added. Otherwise, we cannot be sure we didn't added another bug to this cornerstone method.

Reply:
ok, that explains it